interfaces/builtin/openvswitch.go

```diff
@@ -33,12 +33,32 @@ const openvswitchBaseDeclarationSlots = `
 // sockets needed by ovs-vsctl, ovs-ofctl and ovs-appctl commands. ovs-ofctl
```

Comment:
Does the list of *ctl commands needs to be updated as well?

Reply:
not for the use case I'm writing this change for, as it queries the socket directly.  See https://github.com/greenpau/ovsdb for the details.  The change is to support using https://github.com/greenpau/ovn_exporter for statistics gathering.